src/EventListeners.ts

```diff
@@ -686,7 +686,42 @@ TODO: Fix this entire tab it's utter shit
     DOMCacheGetOrSet(`${String(key)}`).addEventListener('click', (event) => player.blueberryUpgrades[`${String(key)}`].buyLevel(event))
   }
 
+  // BLUEBERRY LOADOUTS
+  for (let i = 1; i <= 8; i++) {
+    // eslint-disable-next-line
+    DOMCacheGetOrSet(`blueberryLoadout${i}`).addEventListener('mouseover', () => createLoadoutDescription(i, player.blueberryLoadouts[i] ?? { ambrosiaTutorial: 0 }))
+    // eslint-disable-next-line
+    DOMCacheGetOrSet(`blueberryLoadout${i}`).addEventListener('click', () => loadoutHandler(i, player.blueberryLoadouts[i] ?? { ambrosiaTutorial: 0 }))
+  }
+  DOMCacheGetOrSet('blueberryToggleMode').addEventListener('click', () => toggleBlueberryLoadoutmode())
+
+  DOMCacheGetOrSet('getBlueberries').addEventListener('click', () => exportBlueberryTree())
   DOMCacheGetOrSet('refundBlueberries').addEventListener('click', () => resetBlueberryTree())
+  // Import blueberries
+  DOMCacheGetOrSet('importBlueberries').addEventListener('change', async e => {
+    const element = e.target as HTMLInputElement
+    const file = element.files![0]
+    let save = ''
+    // https://developer.mozilla.org/en-US/docs/Web/API/Blob/text
+    // not available in (bad) browsers like Safari 11
+    if (typeof Blob.prototype.text === 'function') {
+      save = await file.text()
+    } else {
+      const reader = new FileReader()
+      reader.readAsText(file)
+      const text = await new Promise<string>(res => {
+        reader.addEventListener('load', () => res(reader.result!.toString()))
+      })
+
+      save = text
+    }
+
+    element.value = ''
+    handleLastModified(file.lastModified)
```

Comment:
this is unneeded here. Can be kept once the 2 are combined because it's harmless.